src/licensedcode/models.py

```diff
@@ -877,6 +877,15 @@ def has_unknown(self):
         # TODO: consider using the license_expression_object and the is_unknown
         # license flag instead
         return self.license_expression and 'unknown' in self.license_expression
+    
+    @property
+    def reference_files(self):
```

Comment:
Is this needed? We could directly add `match.rule.referenced_filenames` [here](https://github.com/nexB/scancode-toolkit/pull/2632/files#diff-9b8c9108cb6f529dc48ef0aba7a714c7e2b53feac33c7dc91eac423685a194b7R269). @pombredanne what do you think?

Reply:
oh yeah! you are right !